crypto/ring-signature/src/ring_signature/mlsag.rs

```diff
@@ -112,6 +105,7 @@ impl RingMLSAG {
     // * `check_value_is_preserved` - If true, check that the value of inputs equals
     //   value of outputs.
     // * `rng` - Randomness.
+    #[allow(unreachable_code, unused_variables)]
```

Comment:
Is the purpose of this to allow for streaming?

Reply:
oh hmm, the allow unused was to avoid a compiler error when `alloc` feature is disabled but i changed the approach so may be able to remove it now.

Reply:
Did we determine if we could remove this?